src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -2,80 +2,57 @@ import React, { Fragment } from 'react';
+            {/* eslint-disable-next-line max-len */}
+            <Text style={styles.bold}>{`${nonOptimizedCount}`}</Text><Text>{` of ${totalCount} ${pluralize(totalCount, 'system')} as having a `}</Text><Text style={styles.bold}>non-optimal</Text><Text>{` state.${newLine}`}</Text>
+            {/* eslint-disable-next-line max-len */}
+            <Text style={styles.bold}>{`${staleCount}`}</Text><Text>{` of ${totalCount} ${pluralize(totalCount, 'system')} ${pluralize(staleCount, 'is', 'are')} `}</Text><Text style={styles.bold}>stale*</Text>
+        </Text>
 
-        ]
-    ];
+        <Text style={styles.execInfoText}>Suggestions for stale systems might no longer apply due to systems not being refreshed in 7 days.*</Text>
+    </View>;
 
-    const cpuOccuranceTableData = [
-        [
-            <View key={'cpu-title'} style={{ display: 'flex',
-                flexDirection: 'row' }}>
-                <IconCanvas fillColor='#0066CC'/>
-                <Text>CPU</Text>
-            </View>
+};
 
-        ]
-    ];
+const renderRegisteredSysBreakdown = (data) => {
+    const { non_optimized_count: nonOptimizedCount, non_psi_count: nonPSICount,
+        psi_enabled_count: psiEnabledCount, total_count: totalCount } = data?.meta;
 
     const formattedReportData = formatExecutiveReportData(data);
 
-    const { stateChartData, stateTableData, conditionsChartData,  conditionsTableData, condtionsInfo } = formattedReportData;
+    const { stateChartData, stateTableData } = formattedReportData;
 
-    ioOccuranceTableData.push(...condtionsInfo.io.occurances);
-    ramOccuranceTableData.push(...condtionsInfo.memory.occurances);
-    cpuOccuranceTableData.push(...condtionsInfo.cpu.occurances);
-
-    return <Fragment key="first-page">
-        <Text>
-            This executive summary highlights the performance for your registered systems included in the resource optimization service.
-        </Text>
-        <Text>
-            {`This report gives you an overview of:${newLine}`}
-        </Text>
+    return <View>
+        <Text style={styles.execHeading}>Breakdown of registered systems</Text>
+        {
 
-        <Text>
-            {
-                `${bullet} The number of systems registered
-                ${bullet} Number of registered systems in a non-optimal state
-                ${bullet} Number of system performance issues
-                ${bullet} Description of system performance levels
-                ${bullet} Performance level details for system resources operating in a non-optimal state`}
-        </Text>
+            nonPSICount > 0
+                && <Text>{
+                    /* eslint-disable-next-line max-len */
+                    `${psiEnabledCount} systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information enabled. You could get better suggestions for ${nonPSICount} systems if you enabled Pressure Stall Information. Check the documentation to learn how.`
```

Comment:
Hello @upadhyeammit Thanks for reviewing the PR but I have made the changes since last time, can you please go through the PR now and add any UX text related suggestions in document shared with you on slack please?

Reply:
@upadhyeammit For now I am resolving all the comments, feel free to open any relevant comment or add any new comment if needed. 🙌🏻